### PR TITLE
set disableIsolatedChannels to false in the stress tests

### PR DESCRIPTION
**Pull request by NicholasCouri**
Several stress tests are failing when the stress tests alternate between disableIsolatedChannels == true and false and we end up trying to summarize using the wrong path for the blob/trees.  

Here is Navin's test that hits it: 

1) Summarizer1 with disableIsolatedChannels = true posted summary1 which has 1 data store and 2 DDS.
2) Summarizer2 with disableIsolatedChannels = false loads from summary1 and posts summary2.
3)Summarizer3 with ⁠disableIsolatedChannels = true loads from summary1, changes one of the DDS and tries to post summary3. 4) This summary submission fails.

### CODE REVIEW

File: packages/test/test-service-load/src/optionsMatrix.ts [hunk 1]
```diff
@@ -58,7 +58,7 @@ const gcOptionsMatrix: OptionsMatrix<IGCRuntimeOptions> = {
 };
 
 const summaryOptionsMatrix: OptionsMatrix<ISummaryRuntimeOptions> = {
-    disableIsolatedChannels: booleanCases,
+    disableIsolatedChannels: [false],
```
Comment (agarwal-navin) [inline packages/test/test-service-load/src/optionsMatrix.ts:61]: Maybe just remove it? So it takes whatever is the default.

**Review by anthony-murphy — APPROVED** [review]

File: packages/test/test-service-load/src/optionsMatrix.ts (hunk 1 above)
Comment (anthony-murphy) [reply packages/test/test-service-load/src/optionsMatrix.ts:61]: you cannot remove it. the options matrix requires all fields be set. this is so new options cause a break and must have their values explicitly set.

Comment (agarwal-navin) [reply packages/test/test-service-load/src/optionsMatrix.ts:61]: Ah okay. I wasn't aware of this :-).

**Review by agarwal-navin — APPROVED** [review]

File: packages/test/test-service-load/src/optionsMatrix.ts (hunk 1 above)
Comment (anthony-murphy) [reply packages/test/test-service-load/src/optionsMatrix.ts:61]: you should be able to set the value to undefined, which is essentially default in this case

**Pull request merged**